.devcontainer/setup.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/bash
 
+set -e
+
 env_variables=$(yq eval '.env | to_entries | .[] | .key + "=" + .value' .github/workflows/end2end.yaml | sed 's/\${{[^}]*}}//g') && export $env_variables
 export GIT_ACCESS_TOKEN=${GITHUB_TOKEN}
 export E2E_IMAGE_TAG=latest
 
-array_length=`yq ".runs.steps | length - 1" .github/actions/deploy/action.yaml`
+array_length=$(yq ".runs.steps | length - 1" .github/actions/deploy/action.yaml)
 for i in $(seq 0 $array_length); do
-    step=`yq ".runs.steps[$i]" .github/actions/deploy/action.yaml`
-    working_dir=`yq ".runs.steps[$i].working-directory" .github/actions/deploy/action.yaml`
-    run_command=`yq ".runs.steps[$i].run" .github/actions/deploy/action.yaml`
+    #step=$(yq ".runs.steps[$i]" .github/actions/deploy/action.yaml)
```

Comment:
was this commented on purpose ?

Reply:
yes, I will followup and fix this with TLS...